<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GcomController</class>
 <widget class="QMainWindow" name="GcomController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GcomController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QGroupBox" name="statusGroup">
      <property name="title">
       <string>Status</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="gremlinStatusLabel">
         <property name="text">
          <string>Gremlin Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gremlinStatusField">
         <property name="text">
          <string>GREMLINSTATUS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="gremlinConnectionTime"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="mavlinkPlannerStatusLabel">
         <property name="text">
          <string>MAVLink Relay</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mavlinkStatusField">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Disconnected&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mavlinkStatusMovie">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="mavlinkConnectionTime"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="connectionTab">
       <attribute name="title">
        <string>Connections</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="2">
         <widget class="QGroupBox" name="missionPlannerConnectionGroup">
          <property name="title">
           <string>MAVLink Connection</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="6" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QPushButton" name="mavlinkConnectionButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="mavlinkConnectionStatusField">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="mavlinkIPLabel">
             <property name="text">
              <string>Mission Planner IP</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="mavlinkAutoReconnect">
             <property name="text">
              <string>Auto Reconnect</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="mavlinkPortLabel">
             <property name="text">
              <string>Mission Planner Port</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="mavlinkPortField">
             <property name="text">
              <string>14550</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="mavlinkIPField">
             <property name="text">
              <string>127.0.0.1</string>
             </property>
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="mavlinkConnectionStatusLabel">
             <property name="text">
              <string>Status</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
         <widget class="QGroupBox" name="gremlinConnectionGroup">
          <property name="title">
           <string>Gremlin Connection</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="4" column="1">
            <widget class="QLabel" name="gremlinVersionNumberField">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="gremlinDeviceIDLabel">
             <property name="text">
              <string>Device ID</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="gremlinCapabilitiesLabel">
             <property name="text">
              <string>Capabilities:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="gremlinVersionNumbeLabel">
             <property name="text">
              <string>Version Number</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="gremlinServerPortField"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="gremlinConnectionStatusField">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="gremlinIPVersionLabel">
             <property name="text">
              <string>IP Version</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="gremlinIPAdressField">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="gremlinIPAdressLabel">
             <property name="text">
              <string>IP Address:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="gremlinConnectionStatusLabel">
             <property name="text">
              <string>Socket Status: </string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="gremlinServerIPField"/>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="gremlinIPVersionField">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QPushButton" name="gremlinConnectionButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="gremlinDeviceIDField">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="gremlinServerPortLabel">
             <property name="text">
              <string>Server Port</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QCheckBox" name="gremlinServerAutoResume">
             <property name="text">
              <string>Auto Resume</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="gremlinServerIPLabel">
             <property name="text">
              <string>Server IP</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QListWidget" name="gremlinCapabilitiesField"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="gremlinServerTimeoutLabel">
             <property name="text">
              <string>Timeout (seconds)</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QSpinBox" name="gremlinServerTimeoutField">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>120</number>
             </property>
             <property name="value">
              <number>30</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="IMP">
       <attribute name="title">
        <string>IMP</string>
       </attribute>
       <widget class="QPushButton" name="start_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="imageOutput">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>10</y>
          <width>481</width>
          <height>351</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="update_button">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
       <widget class="QPushButton" name="result_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Get results</string>
        </property>
       </widget>
       <widget class="QPushButton" name="last_button">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>370</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Last</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_button">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>370</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="next_button">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>370</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_button">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>370</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>vmkvgkhjgjkjhkjhhk</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputPath">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>/home/uas/Desktop/GCOM/assets/Samples</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveCSV">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Save CSV</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="outputPath">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>/home/uas/Desktop/GCOM/assets/processed</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>201</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="get_button">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>380</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Get Image</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
